src/tests/e2e/landing.test.e2e.js

```diff
@@ -0,0 +1,63 @@
+import LandingPage from './pages/landing.page'
+import LoginPage from './pages/login.page'
+
+import { sleep } from './testUtils'
+
+const timeout = 180000
+
+describe('Landing page tests', () => {
+  beforeAll(async () => {
+    await page.goto(process.env.E2E_TEST_URL)
+    // login to app
+    const loginPage = new LoginPage()
+    await loginPage.login(
+      process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+      process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD
+    )
+  }, timeout)
+
+  afterAll(async () => {
+    await jestPuppeteer.resetBrowser()
```

Comment:
do we need to reset if using incognito?

Reply:
yes, to ensure this test would not interfere with other tests.